modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/default-user-mgt.xml

```diff
@@ -31,7 +31,7 @@
 		<!--  This is the default configuration -->
 		<UserStoreManager class="org.wso2.carbon.user.core.ldap.ReadWriteLDAPUserStoreManager">
 			<Property name="TenantManager">org.wso2.carbon.user.core.tenant.CommonHybridLDAPTenantManager</Property>
-			<Property name="ConnectionURL">ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}</Property>
```

Comment:
do we use these xml files? These should be removed from the testcases

Reply:
Some of the xmls are not using now. Some of the xmls are being used by test cases which are not in testng file.